zrml/swaps/src/benchmarks.rs

```diff
@@ -395,7 +386,7 @@ benchmarks! {
             false,
             None,
         );
-        let asset_amount: BalanceOf<T> = BASE.saturated_into();
+        let asset_amount: BalanceOf<T> = CENT.saturated_into();
```

Comment:
Now for the release runtime parameters we are below the following.

```rust
bmul(asset_balance.saturated_into(), T::MaxOutRatio::get().saturated_into())?.saturated_into()
```

Nice 💪 . So the reason for this failure was, that the `asset_amount` in this benchmark was set too high, for the actual runtime parameters.